### PR TITLE
Add cite-seq processing section

### LINKED ISSUE #7: Processing Information for CITE-seq
The processing information section will need to be updated to include how we processed CITE-seq data and how we incorporated the CITE-seq counts with RNA-seq counts.

### PR DESCRIPTION
**Pull request by jashapiro**
This PR primarily adds processing information for CITE-seq experiments. 

I tried to keep this concise, but if there are places that you think I should go into more detail, I would be happy to do that. I mostly refer to the previous sections about mapping & quantification, as we use the same overall settings with only minor variations. The one place things differ is the fact that we have no USA-mode equivalent, but I thought I could maybe get away with eliding that. I tried to add some sentences to say that we don't do _that part_ of the RNA-seq pipeline, but I felt like it was more confusing than just leaving it out.

The other major change is relabeling of a header & some wording to use "mapping" instead of "aligning" since we maybe don't quite fully align.
I also added a bit of detail that I felt might have been missing in the discussion of the splici and spliced vs unspliced data. I know that bringing up "ambiguous" transcript counts could be a can of worms, but I thought it worthwhile. Tell me what you think about that. 

Closes #7

### CODE REVIEW

File: docs/processing_information.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ In our hands, we have found that use of the `splici` index led to a more compara
...
 Briefly, selective alignment uses a mapping score validated approach to identify maximal exact matches between reads and the provided index. 
 For all samples, we used selective alignment to the `splici` index. 
 
-A more detailed description of the alignment strategy invoked by `salmon` in conjunction with `alevin-fry` can be found in [Srivastava _et al._ 2020](https://doi.org/10.1186/s13059-020-02151-8).
+A more detailed description of the mapping strategy invoked by `salmon` in conjunction with `alevin-fry` can be found in [Srivastava _et al._ 2020](https://doi.org/10.1186/s13059-020-02151-8).
 
 #### Alevin-fry parameters 
 
-After we aligned FASTQ files using selective alignment to the `splici` index, we continued with the `alevin-fry` pipeline using the following parameters: 
+After we mapping FASTQ files using selective alignment to the `splici` index, we continued with the `alevin-fry` pipeline using the following parameters:
```
Comment (allyhawkins) [inline docs/processing_information.md:28]:
```suggestion
After mapping FASTQ files using selective alignment to the `splici` index, we continued with the `alevin-fry` pipeline using the following parameters: 
```
Just a small wording change for this.

File: docs/processing_information.md [hunk 2]
```diff
@@ -49,8 +51,16 @@ We defined the ambient profile to include all droplets with less than 200 UMI pe
 We consider droplets with an FDR less than or equal to 0.01 to be cell-containing droplets. 
 Only cells that pass this FDR threshold are included in the filtered counts matrix.
 
-### CITE-seq quantification
+## CITE-seq quantification
+
+CITE-seq libraries with reads from antibody-derived tags (ADTs) were also quantified using  [`salmon alevin`](https://salmon.readthedocs.io/en/latest/alevin.html) and [`alevin-fry`](https://alevin-fry.readthedocs.io/en/latest/).
+
+Reference indices were constructed from the submitter-provided list of antibody barcode sequences corresponding to each library using the `--features` flag of `salmon index`.
+Mapping to these indices then proceeded following the same procedures as for RNA-seq data, including mapping with [selective alignment](#selective-alignment) and subsequent [quantification via alevin-fry](#alevin-fry-parameters).
```
Comment (allyhawkins) [inline docs/processing_information.md:59]:
```suggestion
Mapping to these indices followed the same procedures as for RNA-seq data, including mapping with [selective alignment](#selective-alignment) and subsequent [quantification via alevin-fry](#alevin-fry-parameters).
```
This sentence felt a little repetitive to me, so I just cleaned it up a bit.

File: docs/processing_information.md [hunk 3]
```diff
@@ -49,8 +51,16 @@ We defined the ambient profile to include all droplets with less than 200 UMI pe
...
 
-#### Alignment and quantification using alevin-fry
+### Combining CITE counts with RNA counts
 
-#### Combining CITE counts with RNA counts
\ No newline at end of file
+The unfiltered CITE-seq and RNA-seq count matrices often include somewhat different sets of cell barcodes, due to stochastic variation in library construction and sequencing.
+When normalizing these two count matrices to the same set of cells, we chose to prioritize RNA-seq results for broad comparability among libraries with and without CITE-seq data. 
+Any cell barcodes that appeared only in CITE-seq data were discarded, and cell barcodes which did not appear in the CITE-seq data were assigned zero counts for all ADTs.
```
Comment (allyhawkins) [inline docs/processing_information.md:65]:
```suggestion
Any cell barcodes that appeared only in CITE-seq data were discarded.
Any cell barcodes that appeared only in RNA-seq data and did not appear in the CITE-seq data were assigned zero counts for all ADTs. 
```
Something about this sentence was confusing me, so I broke it up into two sentences which I think makes it more clear to show what we did in each of the separate scenarios.

Comment (jashapiro) [reply docs/processing_information.md]: I agree that two sentences is more clear, but I don't like having two sentences that start exactly the same way, so I tried something a bit different.

Comment (allyhawkins) [reply docs/processing_information.md]: Understandable, the change to the sentences that you made look good to me!

**Pull request merged**